setup.cfg

```diff
@@ -0,0 +1,49 @@
+[metadata]
+name = ukrdict
+url = https://github.com/serhii73/ukrdict
+project_urls =
+    CI: Travis = https://travis-ci.com/serhii73/ukrdict
+    GitHub: repo = https://github.com/serhii73/ukrdict
+    GitHub: issues = https://github.com/serhii73/ukrdict/issues
+author = serhii73
+author_email = aserhii@protonmail.com
+description = Library for searching ukrainian words meaning.
+long_description = file: README.rst
+license = MIT
+license_file = LICENSE.rst
+keywords =
+    Ukraine
+    Ukrainian
+    dictionary
+    tool
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
```

Comment:
You should probably add `pypy3` to the Travis CI config: since you advertise compatibility with PyPy it's a good practice to actually test it.